cli/validate_test.go

```diff
@@ -60,35 +60,62 @@ r3rtT0ysHWd7l+Kx/SUCQGlitd5RDfdHl+gKrCwhNnRG7FzRLv5YOQV81+kh7SkU
 -----END RSA PRIVATE KEY-----`
 )
 
-var validateTests = []struct {
-	version       int
-	writeKey      []byte
-	validateKey   []byte
-	expectedError string
+var validateTests = map[string]struct {
+	version               int
+	writeKey              []byte
+	validateKey           []byte
+	expectedCreateError   string
+	expectedValidateError string
```

Comment:
None of the test cases use both error strings. What is the advantage of splitting it in two?

Reply:
Ah, of course, they are after different calls. I should have caught that. Thanks for the correction.